docs/sections/user_guide/cli/tools/mode_template.rst

```diff
@@ -215,6 +186,46 @@ and a YAML file called ``values.yaml`` with the following contents:
      [2023-12-18T23:27:04]    DEBUG ---------------------------------------------------------------------
      [2023-12-18T23:27:04]    DEBUG Read initial values from values.yaml
 
+**NB**: The following examples are based on a ``values.yaml`` file with ``recipient: World`` removed.
+
+* It is an error to render a template without providing all needed values.
+
+  .. code-block:: text
+
+   $ uw template render --input-file template --values-file values.yaml
+   [2024-03-02T16:42:48]    ERROR Required value(s) not provided:
+   [2024-03-02T16:42:48]    ERROR   recipient
+   [2024-03-02T16:42:48]    ERROR Template could not be rendered.
```

Comment:
The code change appears later, but I indented the missing values to stand out better from the surrounding log messages.